src/components/Tooltip/Tooltip.mdx

```diff
@@ -77,13 +98,12 @@ aspect ratio to not render properly.
 
 <Canvas of={TooltipStories.OnLinkedIcons} />
 
-### Tooltip with Icon Content
-
-<Canvas of={TooltipStories.UsingIconContent} />
+### Icon and Image Content
 
-### Tooltip with Image Content
+While the Reservoir Tooltip component does allow icons and images to be included
+in tooltip content, this practice should be avoided.
 
-<Canvas of={TooltipStories.UsingImageContent} />
+<Canvas of={TooltipStories.IconAndImageContent} />
```

Comment:
If we are including examples of things we don't want, maybe we should include a comment over each example:

`// This should be avoided.`

Reply:
Do you mean in the story code?